### ISSUE TITLE
This condition is duplicated in another if/elseif branch

### ISSUE THREAD

**Issue by funivan** [kalessil/phpinspectionsea#550]
```php
$file = 'text_new.html';
if (!is_file($file)) {
  $file = 'text_old.html';
  if (!is_file($file)) { // <-- error
    $file = 'text_fallback.html';
  }
}
```
Hello. I refactor some legacy code. And here is what I find. 
When we check variable, then modify it, and then check again - we get false-positive.

**Comment by kalessil** [comment]
Oh noeeeeesssss, my beloved case =) 
May I ask you to cross-link similar issues here, so I would know which functions to handle in a special way please?

**Comment by funivan** [comment]
@kalessil sure. If I find some similar I will post it.

**Comment by kalessil** [comment]
Similar issues: #381, #339, #289.

**Comment by kalessil** [comment]
Hopefully, I'll deal with the issue in this release.

**Comment by kalessil** [comment]
Fixed!
